src/components/redditClone/newPost.js

```diff
@@ -1,10 +1,23 @@
 import React from 'react'
+import Form from 'components/core/form'
 
 export default props => {
   return (
     <div>
       <div>
-        <button onClick={() => props.onAddPost('https://www.amazon.co.uk', "Earth's Most Customer- centric Company")}>Create a New Post</button>
+        <Form onSubmit={() => props.onAddPost()}>
+          <label htmlFor="link">
+            Link:
+          </label>
+          <input type="link" onChange={e => props.onUpdatePostLink(e.target.value)} value={props.currentlyEditingPost} />
+          <br />
```

Comment:
We don't tend to use `<br />`. You can use `className` on the label, input or form and define a scss style.